model-zoo/src/main/java/ai/djl/basicmodelzoo/cv/classification/MobileNetV2.java

```diff
@@ -0,0 +1,410 @@
+                // add a relu
+                .add(Activation.reluBlock())
+                .add(
+                        Conv2d.builder() // DepthWise
+                                .setKernelShape(new Shape(3, 3))
+                                .setFilters(inputChannels * t)
+                                .optStride(new Shape(stride, stride))
+                                .optPadding(new Shape(1, 1))
+                                .optGroups(inputChannels * t)
+                                .optBias(false)
+                                .build())
+                .add(BatchNorm.builder().optEpsilon(2E-5f).optMomentum(batchNormMomentum).build())
+                .add(Activation.reluBlock())
+                .add(
+                        Conv2d.builder() // PointWise
+                                .setFilters(outputChannels)
+                                .setKernelShape(new Shape(1, 1))
+                                .optBias(false)
+                                .build())
+                .add(BatchNorm.builder().optEpsilon(2E-5f).optMomentum(batchNormMomentum).build());
+
+        // if dimMatch,then add x
+        if (stride == 1 && inputChannels == outputChannels) {
+            block =
+                    new SequentialBlock()
+                            .add(
+                                    new ParallelBlock(
+                                            list ->
+                                                    new NDList(
+                                                            NDArrays.add(
+                                                                    list.get(0).singletonOrThrow(),
+                                                                    list.get(1)
+                                                                            .singletonOrThrow())),
+                                            Arrays.asList(block, Blocks.identityBlock())));
+        }
+        return block; // No reluBlock here
+    }
+
+    /**
+     * Builds a {@link Block} that represent multiple repeats of an inverted residual Unit.
+     *
+     * @param repeat the repeatTimes of an inverted residual Block
+     * @param inputChannels number of inputChannels of the block
+     * @param outputChannels number of outputChannels of the block
+     * @param stride the stride of an inverted residual Unit
+     * @param t the multipleTime of a pointWise Kernel
+     * @param batchNormMomentum the momentum of batchNormLayer
+     * @return a {@link Block} that represent several repeated inverted residual Units
+     */
+    public static Block makeStage(
+            int repeat,
+            int inputChannels,
+            int outputChannels,
+            int stride,
+            int t,
+            float batchNormMomentum) {
+        SequentialBlock layers = new SequentialBlock();
+        layers.add(linearBottleNeck(inputChannels, outputChannels, stride, t, batchNormMomentum));
+        while (repeat - 1 > 0) {
```

Comment:
Can this be changed to a for loop? They are much easier to read then a while loop